.storybook/main.js

```diff
@@ -108,3 +110,25 @@ function overrideDefaultBabelLoader(rules) {
 
   loader.options.customize = customLoaderPath;
 }
+
+/**
+ * @returns {import('storybook-addon-export-to-codesandbox').BabelPluginOptions}
+ */
+function getCodesandboxBabelOptions() {
+  const allPackageInfo = getAllPackageInfo();
+
+  /** @type {import('storybook-addon-export-to-codesandbox').BabelPluginOptions}  */
```

Comment:
can't find a better way to type the `reduce` in the next line... any suggestions welcome

Reply:
you can assert initializer (similar is being done in TS actually):

```
return Object.values(allPackageInfo).reduce(
(acc, cur) => {},
/** @type import('storybook-addon-export-to-codesandbox').BabelPluginOptions*/ ({}))
)
```

 unfortunately there is no way to instantiate generic directly from within function call like `.reduce<ReturnValue>`

Reply:
FML I tried putting the comment above and after the empty object and the only combination I **didn't try was before !!!**